assets/js/modules/globalNav.js

```diff
@@ -144,8 +149,11 @@ define([
      * @returns {Boolean} true if spec shoud be skipped.
      */
     var skipSpec = function(navListCat, obj) {
+        obj = obj || {};
         // obj["cat"] is an array; if cat has needed value
-        var inArray = !!obj["tag"] && obj["tag"].indexOf(navListCat) > -1;
+        var inArray = !!obj["tag"] && typeof(navListCat) === "string"
```

Comment:
No, it can't. This is the internal function which is used with `string` or `string.split` (Array) as an argument.